### PR TITLE
Remove obsoleted rule

### PR DESCRIPTION
**Pull request by scripthunter7**

### CODE REVIEW

File: SocialFilter/sections/general_extensions.txt [hunk 1]
```diff
@@ -232,7 +232,7 @@ crimea-kurort.com#$#img[width="88"][height="31"] {visibility: hidden!important;}
 maskme.ru#$#img[width="88"][height="31"] {visibility: hidden!important;}
 !
 gamesradar.com#$#.shareSocialArticle-wrap { visibility: hidden; }
-softwares-serials.in#$##FTslide { visibility: hidden;!important; }
+softwares-serials.in#$##FTslide { visibility: hidden!important; }
```
Comment (zloyden) [inline SocialFilter/sections/general_extensions.txt:235]: We also add space after the value.
```
softwares-serials.in#$##FTslide { visibility: hidden !important; }
```

Btw, is that domain alive?

**Review by zloyden — CHANGES_REQUESTED** [review]

File: SocialFilter/sections/general_extensions.txt (hunk 1 above)
Comment (scripthunter7) [reply SocialFilter/sections/general_extensions.txt:235]: @zloyden As I see, no :sweat_smile: So I remove the rule instead

Comment (scripthunter7) [reply SocialFilter/sections/general_extensions.txt:235]: I'm just testing the parser

Comment (scripthunter7) [reply SocialFilter/sections/general_extensions.txt]: Done

**Review by zloyden — APPROVED** [review]

**Pull request merged**